app/controllers/management/base_controller.rb

```diff
@@ -32,9 +32,10 @@ def managed_user
     end
 
     def check_verified_user(alert_msg)
-      unless managed_user.level_two_or_three_verified?
-        redirect_to management_document_verifications_path, alert: alert_msg
-      end
+      return if managed_user.persisted? && managed_user.level_two_or_three_verified?
+
+      message = !managed_user.persisted? ? t("management.need_managed_user") : alert_msg
```

Comment:
```suggestion
      message = managed_user.persisted? ? alert_msg : t("management.need_managed_user")
```

So we avoid the `if not ... else ...` pattern which makes Rai cry :smile:.